runtime/web_worker.rs

```diff
@@ -398,7 +398,7 @@ impl WebWorker {
     });
 
     // NOTE(bartlomieju): ordering is important here, keep it in sync with
-    // `runtime/build.rs`, `runtime/worker.rs` and `runtime/snapshot.rs`!
+    // `runtime/worker.rs` and `runtime/snapshot.rs`!
 
     let mut extensions = vec![
       // Web APIs
```

Comment:
Factor out to a separate helper in `shared.rs` so this list doesn't have to be kept in sync.